routing/index_graph_starter_joints.cpp

```diff
@@ -226,30 +230,31 @@ void IndexGraphStarterJoints::GetEdgeList(JointSegment const & vertex, bool isOu
 
   if (!isOutgoing)
   {
-    // |parentSegment| - это вершина-родитель от которой мы ищем детей
-    // Надо взять вес joint'a, который оканчивается в |parentSegment|
+    // |parentSegment| - it is parent-vertex, from which we searches children.
```

Comment:
|parentSegment| is parent-vertex from which we search children

Reply:
done